ctapipe_io_lst/calibration.py

```diff
@@ -205,6 +210,13 @@ def update_first_capacitors(self, event: ArrayEventContainer):
             )
 
     def calibrate(self, event: ArrayEventContainer):
+
+        if ((event.trigger.event_type == EventType.FLATFIELD or
+             event.trigger.event_type == EventType.SKY_PEDESTAL) and
+            not self.calibrate_ff_and_ped):
```

Comment:
I don't really like that you convert this method into a no-op depending on this options.

I think it would be better to move the traitlet to the source and then do:

```
if self.calibrate_ff_and_ped or event.trigger.event_type == EventType.SUBARRAY:
    self.lstr0corrections.calibrate(event)
```

Reply:
ok for me, but are we sure there are not other triggers we like to calibrate?

Reply:
Sorry, the proposal was mainly to move it to the source, not in here. If you have a more appropriate check (like explictly checking for the pedestals or flatfield) go ahead.

Reply:
Ok, I move it there (I was indeed undecided)